### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Name property is empty for "Create App Service dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1397]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app.
4. "Deploy web app" dialog will open. 
5. In "Deploy web app" dialog , navigate to and select "Create" button.
6. Observe the name property for the dialog.

**Actual Result:**
Name property is empty for the Dialog.

**Narrator Behaviour:**
Create..., button,
Enter
Enter name, editing webapp-180129114655,  

**NVDA:** NVDA is announcing as expected
Create App Service
Create Azure App Service
Enter name  edit
webapp-180129122934

**Expected Result:**
Name property should be "Create App Service" dialog

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![create app service-dialog](https://user-images.githubusercontent.com/22956130/35684029-ac1b5ec6-078b-11e8-942f-cb8888881860.jpg)
![create app service-dialog-nvda](https://user-images.githubusercontent.com/22956130/35684031-af7313d4-078b-11e8-9224-3f717881381b.jpg)

**Comment by Eskibear** [comment]
ref  #1407

**Comment by jdneo** [comment]
Take Eclipse's official TitleAreaDialog as an example:

![image](https://user-images.githubusercontent.com/6193897/38715085-c95a855e-3f0b-11e8-9db8-95bde333ff99.png)

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by jdneo** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can still see the property in the `Value` field
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

Same with  #1410 
![image](https://user-images.githubusercontent.com/22956130/35684031-af7313d4-078b-11e8-9224-3f717881381b.jpg)

**Comment by jdneo** [comment]
Can show the name property on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0

![image](https://user-images.githubusercontent.com/6193897/47150146-4673e680-d308-11e8-8d87-62732ecc3256.png)

**Comment by Tanushagottipati** [comment]
This issue is still reproducing in the current test pass

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749840
